app/src/main/java/org/oppia/android/app/settings/profile/ProfileEditViewModel.kt

```diff
@@ -23,26 +23,26 @@ class ProfileEditViewModel @Inject constructor(
 
   private val isAllowedDownloadAccessMutableLiveData = MutableLiveData<Boolean>()
 
-  /** The download access for the profile. */
+  /** Download access enabled for profile by the administrator. */
   val isAllowedDownloadAccess: LiveData<Boolean> = isAllowedDownloadAccessMutableLiveData
 
-  /** The list of all the current profiles registered in the app [ProfileListFragment]. */
+  /** List of all the current profiles registered in the app [ProfileListFragment]. */
   val profile: LiveData<Profile> by lazy {
     Transformations.map(
       profileManagementController.getProfile(profileId).toLiveData(),
       ::processGetProfileResult
     )
   }
 
-  /** The admin rights for the profile. */
+  /** Whether the user is admin. */
   var isAdmin = false
 
-  /** The internal profile id of the profile. */
+  /** Internal profile id of the profile. */
   fun setProfileId(id: Int) {
     profileId = ProfileId.newBuilder().setInternalId(id).build()
   }
 
-  /** This fetches the profile of a user asynchronously. */
+  /** This method fetches the profile of a user asynchronously. */
```

Comment:
```suggestion
  /** Fetches the profile of a user asynchronously. */
```

The summary fragment of a method KDoc reads as though it starts with "This method" but shouldn't actually include that prefix.